kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3OutputStream.java

```diff
@@ -103,14 +106,17 @@ private void uploadPart() throws IOException {
     buffer.clear();
   }
 
-  private void uploadPart(int size) throws IOException {
+  private void uploadPart(final int size) throws IOException {
```

Comment:
Due to the fact that this argument is later on passed into the inner class (Runnable) it has to stay final.